src/i18n-keysets/component.dialog-relations.view/en.json

```diff
@@ -26,6 +26,7 @@
   "label_charts": "charts",
   "label_choose-field-for-alias": "Choose field for alias",
   "label_choose-fields-from-list": "Choose fields from list",
+  "label_current-selector": "Selector: ",
```

Comment:
It's a bad practice to add space and colon inside keysets! Need to remove it and add it on usage